src/main/helpers.js

```diff
@@ -16,6 +16,7 @@
 
 import stream from 'stream'
 import mime from 'mime-types'
+var Crypto = require('crypto')
```

Comment:
can we do `import Crypto from 'crypto'`?

Reply:
This imports based on browserify environment. So this is needed. For md5 to to be browserified and work in browser environment.